packages/csv-parser-price/test/main.spec.js

```diff
@@ -51,8 +51,11 @@ describe('CsvParserPrice::parse', () => {
 
     const outputStream = streamtest['v2'].toText((error, result) => {
       const prices = JSON.parse(result).prices
+      const price = prices[0].prices[0]
```

Comment:
So the actual price is in the second `prices`, majorly because this is what is expected from the node-cli